plugins/modules/docker_compose_v2.py

```diff
@@ -0,0 +1,923 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright Léo El Amri
```

Comment:
Please note that in case you copied significant parts of an existing module (like documentation), you need to add the copyright lines from that module here as well.